### DISCUSSION TITLE
Stuck at loading when creating a new list

### DISCUSSION THREAD

**Discussion by bwcr** [community/community#114445] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/50513707/9d994140-aa48-4008-a9eb-ffcc7680460c)
My internet connection is fine, but I am having trouble creating a new list as it seems to be stuck on the loading state.

**Comment by a-god-of-death** [comment]
Yes, same here

**Reply by VahidRafieiPour** [reply]
1000btc

**Comment by ivrolan** [comment]
I came here just to post the same. I think Lists are such a powerful tool to save repos and not losing them over time. Thank you for them!

**Reply by Sokhanetaze80** [reply]
من این کد رو چطور در آرون اسکن اجراکنم؟

**Comment by sarvan2research** [comment]
i too faced same issue, tried to refresh the github url, list values got updated.

**Comment by Jv2350** [comment]
Check Network Connection: Ensure that your internet connection is stable. A slow or unstable connection can sometimes cause loading issues.
Clear Browser Cache: Clear your browser's cache and cookies. Sometimes, old data stored in the cache can interfere with the loading of new content.
Try a Different Browser: If clearing the cache doesn't help, try using a different web browser to see if the issue persists. This can help determine if the problem is specific to your current browser.

**Comment by mwoss** [comment]
Same issue here. It created the list underneath, but the modal was still visible.

**Comment by Ketleypratessilva** [comment]
Olá bom dia todo para eu sou surda - min ajudar por favor 
- [ ] Quero aprender. _Eu tenho foto logo_
- [ ] Sim 
- [ ] Não

**Comment by saeedbamri** [comment]
https://github.com/orgs/community/discussions/125235

**Comment by arfogg** [comment]
Just to say this is happening for me too. Here's a screenshot of the freeze:
![image](https://github.com/user-attachments/assets/c0be8283-c2e7-4764-8503-b079067ba42d)
I'm on the github webpage in Chrome v127.0.6533.100

Fix for me is refreshing the page, and then the list has in fact been created.
![image](https://github.com/user-attachments/assets/e91afe4d-09fe-4561-bc7e-415f623d382a)

On the comment above, my internet connection is stable and speed:
Download: 174.88 Mbps
Upload: 214.78 Mbps

**Reply by MarketBeatAd** [reply]
Have you ever tried to use links or share them here or so so in other places

**Reply by arfogg** [reply]
hey @MarketBeatAd , I'm not sure what you mean. Is there anything specific you need a link to I can add here?

**Comment by AndreyVladimirovish** [comment]
how can I earn my own capital

**Comment by irisdyoung** [comment]
Hey, I think this symptom is not a loading issue but an issue of cursor focus. (See some of the other bug reports describing not being able to type in the fields when creating a new list.) If you experience not being able to interact with the popup, but clicking will click _through_ to whatever is behind it (which is what tipped me off), it's a focus issue.

Also, I found that I have this issue when I'm creating a new list from the context menu in a repo I want to add to a new list, but not if I navigate to my own profile > lists > create new list there. If I create the empty list I can then go back to the repo and add it to the list. Hope this helps!

**Reply by Ketleypratessilva** [reply]
@MarketBeatAd

**Comment by Nagafi** [comment]
😍🙏🙏🙏🙏

**Comment by astro-com-my** [comment]
![Screenshot_1](https://github.com/user-attachments/assets/a8efbb9f-8580-4cd4-b4d6-989a003cb18e)

Same here tried to push to origin with github desktop. Stuck at "Waiting for a runner to pick up this job...". Please help!

**Comment by pingpr0** [comment]
Не хуя вообще не понятно 🤣🤣🤣
